### ISSUE TITLE
[apollo-server-hapi]

### ISSUE THREAD

**Issue by creativehubspace** [apollographql/apollo-server#3397]
## Update dependencies
I got the following warnings after installing the apollo-server-hapi package:

* boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
* accept > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.

Could they please be updated.

**Comment by kachkaev** [comment]
Same in `apollo-server-micro`. Seeing this during `yarn install`:

```
warning apollo-server-micro > accept@3.1.3: This module has moved and is now available at @hapi/accept. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning apollo-server-micro > accept > boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning apollo-server-micro > accept > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning apollo-server-micro > accept > boom > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
```

**Comment by glasser** [comment]
My suspicion is that we fixed this in Apollo Server 3 when we upgraded Hapi a lot.

That said, Apollo Server 4 replaces a hard-coded set of web framework integrations with a simple stable API for building integrations. As part of this, the core project no longer directly supports a Hapi integration. It should not be that hard for excited Hapi users to build and maintain a Hapi integration, and our guess is they will do a better job of it than the Apollo Server core team, who did not use Hapi at all other than to maintain the integration. There is not a Hapi integration at this time, but some interest has been expressed over at https://github.com/apollographql/apollo-server/issues/6028. Sorry for the delay!
